obspy/io/nlloc/tests/test_core.py

```diff
@@ -351,6 +351,14 @@ def test_reading_nlloc_v7_hyp_file(self, testdata):
         assert cat[0].origins[0].arrivals[0].azimuth == 107.42
         # compare test_rejected_origin test case
         assert cat[0].origins[0].evaluation_status is None
+        assert cat[0].origins[0].uncertainty.confidence_ellipsoid is not None
```

Comment:
It looks like this line is failing the CI. Appart from whatever real fix is needed, perhaps best to change to: 
```python
assert getattr(cat[0].origins[0].uncertainty, "confidence_ellipsoid", None) is not None
```
so it will properly raise an `AssertionError` and fail the test rather than raise an `AttributeError`.

Reply:
Attribute name is wrong.. you need to look up `cat[0].origins[0].origin_uncertainty`